designs/shelly/src/back.mjs

```diff
@@ -0,0 +1,152 @@
+import { pluginBundle } from '@freesewing/plugin-bundle'
+import { base } from './base.mjs'
+
+function draftBack({
+  utils,
+  Path,
+  Point,
+  paths,
+  points,
+  measurements,
+  options,
+  part,
+  store,
+  paperless,
+  complete,
+  sa,
+  macro,
+  snippets,
+  Snippet,
+}) {
+  const raglanAngle = store.get('raglanAngle')
+  const neckRadius = store.get('neckRadius')
+
+  points.neckCenter = points.raglanCenter.shift(270, -options.neckBalance * neckRadius)
+
+  points.neckShoulderCorner = utils.beamIntersectsCircle(
+    points.neckCenter,
+    neckRadius,
+    points.raglanCenter,
+    points.armholeCorner
+  )[1]
+
+  const neckShoulderRadius = points.raglanCenter.dist(points.neckShoulderCorner)
+  store.set('backNeckRadius', neckShoulderRadius)
+
+  points.cfNeck = points.neckCenter.shift(270, neckRadius)
+
+  const necklineAngleAtRaglan = points.cfNeck.angle(points.neckShoulderCorner) * 2
+  const necklineArcLength = utils.deg2rad(necklineAngleAtRaglan) * neckRadius
+  points.neckCP1 = points.neckShoulderCorner.shift(
+    necklineAngleAtRaglan + 180,
+    necklineArcLength / 3
+  )
+  points.neckCP2 = points.cfNeck.shift(0, necklineArcLength / 3)
+
+  const backNecklineToRaglanAngle = raglanAngle - (necklineAngleAtRaglan + 180)
+  store.set('backNecklineToRaglanAngle', backNecklineToRaglanAngle)
+
+  const drawSide = () => {
```

Comment:
Avoid defining functions in your draft method as they will be re-created on each draft.
Instead, move them out of the function body.